regression/contracts/quantifiers-loop-01/main.c

```diff
@@ -0,0 +1,20 @@
+#include <assert.h>
+
+void main()
+{
+  int N = 64, a[64];
+
+  for(int i = 0; i < N; ++i)
+    // clang-format off
+    __CPROVER_loop_invariant(
+      (0 <= i) && (i <= N) &&
+      __CPROVER_forall {
+        int k;
+        (0 <= k && k < N) ==> 1 == 1
```

Comment:
Is the `1 == 1` so that it could simplify away completely intentional?

Reply:
Shouldn't the invariant be the following?
```c
      __CPROVER_forall {
         int k;
         (0 <= k && k < N) ==> a[k] = 1
      }
```

Reply:
`(0 <= k && k < N) ==> a[k] = 1` wouldn't hold, but I am adding `(0 <= k && k < i) ==> a[k] = 1` to address this concern.